### PR TITLE
Adding deprecated to the living application layout display name

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [BPO-62](https://bonitasoft.atlassian.net/browse/BPO-62)

### CODE REVIEW

**Comment by DumitruCorini** [comment]
migration done at [#249](https://github.com/bonitasoft/bonita-migration-sp/pull/249)

File: uid-pages/layout-living-application/src/resources/page.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #The name must start with 'custompage_'
 #Mon Feb 25 17:37:11 CET 2019
 name=custompage_defaultlayout
-displayName=Default living application layout
-description=This is the default layout V5 definition for a newly-created application. It was created using the UI designer, so you can export it and edit it with the UI designer. It contains a horizontal menu widget and an iframe widget. The menu structure  is defined in the application navigation. The application pages are displayed in the iframe.
```
Comment (abirembaut) [inline uid-pages/layout-living-application/src/resources/page.properties]: Maybe "This layout is deprecated. Use the Bonita Layout instead."

Comment (DumitruCorini) [reply uid-pages/layout-living-application/src/resources/page.properties]: done

**Comment by abirembaut** [comment]
LGTM

**Pull request merged**